src/components/transport_manager/src/transport_manager_default.cc

```diff
@@ -139,6 +143,18 @@ int TransportManagerDefault::Init(resumption::LastState& last_state) {
   AddTransportAdapter(ta_cloud);
 #endif  // CLOUD_APP_WEBSOCKET_TRANSPORT_SUPPORT
 
+#ifdef WEBSOCKET_SERVER_TRANSPORT_SUPPORT
+  auto ta_websocket = new transport_adapter::WebSocketTransportAdapter(
```

Comment:
I would prefer the naming `WebsocketServerTransportAdapter` for the new TA class

Reply:
@JackLivio Fixed in b0b1c52ead4d9178d0393f0f2c6ebd2b75703f8d